entities/package.py

```diff
@@ -241,6 +268,7 @@ def __str__(self) -> str:
             ' {}\n'.format(self.name) +\
             ' {}\n'.format(self.description) +\
             ' {} {}\n'.format(self.position_3d, self.rotation_3d)
+        ret += indent_entities(self.models_3d)
```

Comment:
Should be sorted, just in case some packages will have multiple 3D models:

```suggestion
        ret += indent_entities(sorted(self.models_3d))
```

Reply:
True, but this requires implementing `__eq__` and `__lt__` on the entity class.